src/lib/Hydra/Controller/Build.pm

```diff
@@ -66,9 +66,11 @@ sub build_GET {
 
     $c->stash->{template} = 'build.tt';
     $c->stash->{isLocalStore} = isLocalStore();
+    # XXX: If ca-derivations is enabled then this will always return false
+    # because `$_->path` will be empty
```

Comment:
Is this always true, or is it true if it is set AND the derivation is ca-addressed? This feels like a fairly spooky change to the data model of Hydra.

Reply:
Mh… I see what you mean. It's only true if the derivation is ca. I've updated the comment in ca86783e9d9c5e6fe744bfd33dc9846122203935 to make it clearer.

> This feels like a fairly spooky change to the data model of Hydra.

It is indeed :smiley: . But the `path` must be nullable, because of the very nature of CA derivations